src/unicycle_2d/ignition.cpp

```diff
@@ -88,17 +88,21 @@ void Ignition::onInit()
   // Connect to the reset service
   if (!params_.reset_service.empty())
   {
-    reset_client_ = node_handle_.serviceClient<std_srvs::Empty>(params_.reset_service);
+    reset_client_ = node_handle_.serviceClient<std_srvs::Empty>(ros::names::resolve(params_.reset_service));
```

Comment:
Could you do the same for the server side in `fuse`, please?

Reply:
Sure.